js/load-image-scale.js

```diff
@@ -38,9 +38,17 @@
   }
 
   loadImage.transform = function (img, options, callback, file, data) {
+    var scaleImg
+
+    try {
+      scaleImg = loadImage.scale(img, options, data)
+    } catch (error) {
+      scaleImg = error
+    }
+
```

Comment:
@blueimp 
Do you think this is a good place to handle uncaught exceptions?
Im considering maybe it's better to handle it for all in the original transform 🤔